wagtail/wagtaildocs/static_src/wagtaildocs/scss/add-multiple.scss

```diff
@@ -0,0 +1,91 @@
+@import 'wagtailadmin/scss/variables';
+@import 'wagtailadmin/scss/mixins';
+@import 'wagtailadmin/scss/grid';
+
+.replace-file-input {
+    display: inline-block;
+    position: relative;
+    overflow: hidden;
+    padding-bottom: 2px;
+
+    input[type=file] {
+        padding: 0;
+        opacity: 0;
+        position: absolute;
+        top: 0;
+        right: 0;
+        direction: ltr;
+        width: auto;
+        display: block;
+        font-size: 5em;
+
+        &: hover {
```

Comment:
The whitespace after `&:` (here and immediately below) should be removed - it breaks on my copy of node-sass.